src/js/angular/core/templates/rdfResourceSearchTemplate.html

```diff
@@ -0,0 +1,25 @@
+<link href="css/rdf-resource-search.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
```

Comment:
I don't like how in general the versionning is implemented right now. It forces us to do such things like the above instead of just importing the stylesheet in the directive like this
```javascript
import './rdf-resource-search.css';
```